dev/components/menu/tpl.pug

```diff
@@ -1,17 +1,25 @@
 .app-menu(ng-class="vm.className")
-  .menu-button(ng-click="vm.redirect('/dashboard')")
+  .menu-button(
+    ng-click="vm.redirect('/dashboard')"
+    ng-class="vm.dashboardClass")
```

Comment:
Please use 'vm.url' to check active link eg:
ng-class="{active: vm.url === '/conversations'}"